XPlat/Shopify/Android/nuget/Xamarin.Shopify.Android.nuspec

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Xamarin.Shopify.Android</id>
+    <title>Shopify Mobile Buy SDK for Xamarin.Android</title>
+    <version>$version$</version>
+    <authors>Xamarin Inc.,Shopify Inc.</authors>
+    <owners>Xamarin Inc.</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Shopify’s Mobile Buy SDK makes it simple to sell physical products inside your mobile app. With a few lines of code, you can connect your app with the Shopify platform and let your users buy your products using their credit card.</description>
+    <summary>Sell with Shopify in Android apps</summary>
+    <copyright>Copyright © Xamarin. All Rights Reserved.</copyright>
+    <licenseUrl>https://raw.githubusercontent.com/Shopify/mobile-buy-sdk-android/$version$/LICENSE</licenseUrl>
```

Comment:
I would use the Component License URL.  Also set the project URL to the component location.